.github/workflow/tests.yml

```
@@ -0,0 +1,27 @@
+name: Test ocrd_cis installation and run tests
+
+on:
+  push:
+    branches: [ "master" ]
+  pull_request:
+    branches: [ "master" ]
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      fail-fast: false
+      matrix:
+        python-version: [ "3.8", "3.9", "3.10", "3.11", "3.12" ]
+        os: [ "ubuntu-22.04" ]
+
+    steps:
+      - uses: actions/checkout@v4
+      - name: Set up Python ${{ matrix.python-version }}
+        uses: actions/setup-python@v5
+        with:
+          python-version: ${{ matrix.python-version }}
+      - name: Install ocrd_cis
+        run: make install
+      - name: Test ocrd_cis
+        run: make test
```

We will need a JRE for the aligner and postprocessor part:
```suggestion
      - uses: actions/setup-java@v4
        with:
          distribution: 'zulu'
          java-version: '11'
      - name: Install ocrd_cis
        run: make install
      - name: Test ocrd_cis
        run: make test V=""
```

---

Ops, you are right. Thanks!